src/views/Teams/EditTeam.jsx

```diff
@@ -0,0 +1,34 @@
+import React, {useState} from 'react'
+import TeamForm from '../../components/TeamForm'
+import { useHistory } from 'react-router';
+import { updateTeamById } from '../../services/teams';
+import { useParams } from 'react-router';
+
+export default function EditTeam() {
+    const [teamName, setTeamName] = useState('');
+    const [teamCity, setTeamCity] = useState('');
+    const [teamState, setTeamState] = useState('');
+    const history = useHistory();
+    const {id} = useParams();
+    console.log(id);
```

Comment:
If you want to preload the inputs with the team's original info you can use a useEffect to get the team by id, and then set the name, city, and teamState to state. The inputs in the form would need value changed to defaultValue to be able to edit them.